src/views/Datasets.vue

```diff
@@ -241,8 +256,15 @@ export default {
 				this.isBusy = false
 			}
 		},
-		open(id) { // should maybe later be changed to link so that accessability is better
-			this.$router.push({ name: 'editor', params: { id: id }})
+		editDataset(item) {
+			this.$router.push({ name: 'editor', params: { id: item.id }})
+		},
+		clickedRow(item, fuu) {
+			item._showDetails = !item._showDetails
+		},
+		doNothing(e) {
```

Comment:
Not used anywhere. I think you already found out that there are .stop and .prevent

Reply:
removing

Reply:
should be fixed